l2-frontend/src/fields/AnesthesiaProcess.vue

```diff
@@ -265,7 +266,7 @@
           'research_data': research_data
         });
         this.tb_data = [...data.data];
-        console.log(this.tb_data)
+        this.row_category = {...data.row_category}
```

Comment:
А зачем тут spread делать?
Он тут не нужен, достаточно просто
```suggestion
        this.row_category = data.row_category
```

так как this.row_category даже меняться нигде не будет после присваивания